process/tests/fuser/mod.rs

```diff
@@ -110,27 +97,42 @@ fn get_username_by_uid(uid: uid_t) -> io::Result<String> {
 /// - Verifies that the owner printed in stderr.
 #[test]
 fn test_fuser_with_user() {
-    let process = Command::new("sleep")
-        .arg("1")
-        .spawn()
-        .expect("Failed to start process");
-
-    let pid = process.id();
+    thread::spawn(move || {
+        let temp_file_path = "/tmp/test_fuser_with_user";
+        File::create(temp_file_path).expect("Failed to create temporary file");
+
+        let mut process = Command::new("tail")
+            .arg("-f")
+            .arg(temp_file_path)
+            .spawn()
+            .expect("Failed to start process");
+
+        let pid = process.id();
+
+        wait_for_process(pid);
+
+        thread::sleep(Duration::from_millis(100));
```

Comment:
Why do we need a sleep after we've already waited for the process to spawn?

Reply:
If that's needed to make sure tail started reading the file - that's very unreliable.